src/blog/NewsContainer.test.tsx

```diff
@@ -30,26 +30,23 @@ const testCategories = [
 
 describe(NewsContainer.name, () => {
   it('should filter articles by filter when category button is clicked', () => {
-    const instance = mount(
+    render(
       <NewsContainer articles={testArticles} categories={testCategories} />
     )
-
-    const researchCategoryFilterButton = instance.find('[type="radio"]').last()
-    researchCategoryFilterButton.simulate('click')
-    expect(instance.find('NewsCard').length).toBe(2)
-
-    const generalCategoryFilterButton = instance.find('[type="radio"]').first()
-    generalCategoryFilterButton.simulate('click')
-    expect(instance.find('NewsCard').length).toBe(pageSize)
+    const researchCategoryFilter = screen.getByLabelText('Research')
```

Comment:
These `getBy*` etc methods are available for use from the `render` return (i.e. `const { getByLabelText } = render(</>)`) and this is our preferred way of implementation - could we swap these to use the same approach as the other test files, rather than importing the `screen` directly? 👼  It's also faster as you're only searching the component, rather than the entire DOM.

Reply:
Thanks for the advice, Em! I've made the changes now